app/controllers/practices_controller.rb

```diff
@@ -183,43 +182,6 @@ def search
     @practices = Practice.searchable_practices
     @facilities_data = facilities_json
     @practices_json = practices_json(@practices)
-
-  end
-
-  def planning_checklist
-    @facilities_data = facilities_json
-  end
-
-  # GET /practices/1/committed
-  def committed
-    render 'committed'
-  end
-
-  # POST /practices/1/commit
-  # POST /practices/1/commit.json
-  def commit
-    user_practice = UserPractice.find_by(user: current_user, practice: @practice, committed: true)
-
-    if user_practice.present?
-      flash[:notice] = "You have already committed to this practice. If you did not receive a follow-up email from the practice support team yet, please contact them at #{@practice.support_network_email || ENV['MAILER_SENDER']}"
-    else
-      user_practice = UserPractice.find_or_initialize_by(user: current_user, practice: @practice)
-      user_practice.committed = true
-      user_practice.time_committed = DateTime.now
-      PracticeMailer.commitment_response_email(user: current_user, practice: @practice).deliver_now
```

Comment:
Because we aren't using `PracticeMailer` class anywhere else now we can remove the `app/mailers/practice_mailer.rb` file :)

Reply:
Done